README.md

```diff
@@ -39,7 +39,7 @@ More information on contributing and the general code of conduct for discussion
 ## List of Scripts in Repo
 
 | Script                                   | Link                                                                                                                                                     | Description                                                                                                                                                       |
-| ---------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Could you remove this changes? There should be only new item adding part. Not the other changes.

Reply:
Yes. sorry about that. dropped.